.travis.yml

```diff
@@ -15,7 +15,7 @@ env:
 
 before_install:
   - sudo apt-get -qq update
-  - sudo apt-get install -y avahi-daemon libavahi-client-dev libavahi-common-dev libavahi-common3 libdbus-1-dev libupnp-dev libespeak-dev
+  - sudo apt-get install -y avahi-daemon libavahi-client-dev libavahi-common-dev libavahi-common3 libdbus-1-dev libupnp-dev libespeak-dev libudev-dev
```

Comment:
we don't need it for OSX :) (udev is purely a linux thing). On OSX it uses some CoreFoundation and IOKit stuff that I don't really know.